fuel-txpool/src/txpool.rs

```diff
@@ -110,6 +120,30 @@ impl TxPool {
         }
         Vec::new()
     }
+
+    fn verify_tx_min_gas_price(&mut self, tx: &Transaction) -> Result<(), Error> {
+        if tx.gas_price() < self.config.min_gas_price {
+            return Err(Error::NotInsertedGasPriceTooLow);
+        }
+        Ok(())
+    }
+
+    fn verify_tx_min_byte_price(&mut self, tx: &Transaction) -> Result<(), Error> {
+        if tx.byte_price() < self.config.min_byte_price {
+            return Err(Error::NotInsertedBytePriceTooLow);
+        }
+        Ok(())
+    }
+
+    fn verify_tx_min_byte_price_ratio(&mut self, tx: &Transaction) -> Result<(), Error> {
+        if self.config.max_gas_per_byte_price > 0
+            && tx.byte_price() * self.config.max_gas_per_byte_price < tx.gas_price()
```

Comment:
maybe we should update fuel-spec to address this, it is currently set as `bytePrice != gasPrice * GAS_PER_BYTE`
`

Reply:
Removed this since configurable min fees should be sufficient for now. We can revisit multi-modal pricing once we have legit block production and more interesting resource contention issues.